src/Validation/StarterKitValidation.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace AcquiaCMS\Cli\Validation;
+
+use JsonSchema\Validator;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Validation service to validate each key in starter kit.
+ */
+class StarterKitValidation {
+
+  /**
+   * Holds the symfony console output object.
+   *
+   * @var \Symfony\Component\Console\Output\OutputInterface
+   */
+  protected $output;
+
+  /**
+   * Constructs an object.
+   *
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *   Returns an absolute root path to project.
+   */
+  public function __construct(
+    OutputInterface $output) {
+    $this->output = $output;
+  }
+
+  /**
+   * Loop each starter-kit and send it for validation.
+   */
+  public function validateStarterKits(array $schema, array &$starterkits): void {
+    foreach ($starterkits as $starterkit) {
+      $this->validateStarterKit($schema, $starterkit);
+    }
+  }
```

Comment:
```suggestion
  public function validateStarterKits(array $schema, array &$starterkits): void {
    foreach ($starterkits as $name => $starterkit) {
      $this->errorMessage = $this->errorMessage . $this->validateStarterKit($schema, $starterkit, $name);
    }
    if ($this->errorMessage) {
      throw new AcmsCliException(PHP_EOL . $this->errorMessage . PHP_EOL);
    }
  }
```
<img width="902" alt="Screenshot 2023-08-21 at 3 00 15 PM" src="https://github.com/acquia/acquia-cms-starterkit/assets/19570710/41252eee-ecc4-4ab8-ba1e-c74d4cf947b3">